proto/cmp/types/v1alpha/contact_info.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha;
+
+import "cmp/types/v1alpha/address.proto";
+import "cmp/types/v1alpha/phone.proto";
+import "cmp/types/v1alpha/email.proto";
+import "cmp/types/v1alpha/link.proto";
+
+// Contact Info for general use.
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/contact_info.proto.dot.xs.svg)
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/contact_info.proto.dot.svg)
+
+
+message ContactInfo {
+
+    // Address
+    repeated cmp.types.v1alpha.Address address = 1;
```

Comment:
```suggestion
// Contact Info for general use.
//
// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/contact_info.proto.dot.xs.svg)
// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/contact_info.proto.dot.svg)
message ContactInfo {

    // Address
    repeated cmp.types.v1alpha.Address address = 1;
```

Comments should stick to the message declaration. Otherwise they are not considered a comment for that message and also not rendered in the buf.build website.